bridge/svix-bridge/src/webhook_receiver/mod.rs

```diff
@@ -181,5 +197,161 @@ async fn transform(
+            .await
+        {
+            Ok(resp) => {
+                tracing::trace!(count = resp.data.len(), "got messages");
+                for msg in resp.data.into_iter() {
+                    let payload = match parse_payload(
+                        &SerializablePayload::Standard(
+                            // FIXME: for svix-event pollers we already know the payload is json so
+                            //   there's some wasted ser/deser/ser cycles.
+                            serde_json::to_vec(&msg)
+                                .expect("just fetched as json, must be serializable"),
+                        ),
+                        poller.transformation.as_ref(),
+                        poller
+                            .transformer_tx
+                            .clone()
+                            .expect("transformer tx is required"),
+                    )
+                    .await
+                    {
+                        Err(status) => {
+                            tracing::error!(
+                                status = status.as_u16(),
+                                "error while parsing polled message"
+                            );
+                            // BACKOFF
+                            sleep_time = (sleep_time * 2).clamp(MIN_SLEEP, MAX_SLEEP);
+                            // Retry the current iterator.
+                            continue;
+                        }
+                        Ok(p) => p,
+                    };
+                    if let Err(status) = handle(payload, poller.output.clone()).await {
+                        // FIXME: need to refactor handle to not give http status codes so we can report what happened here.
+                        tracing::error!(
+                            status = status.as_u16(),
+                            "error while handling polled message"
+                        );
+                        // BACKOFF
+                        sleep_time = (sleep_time * 2).clamp(MIN_SLEEP, MAX_SLEEP);
+                    }
+                }
+                tracing::trace!(
+                    ?iterator,
+                    next_iterator = ?resp.iterator,
+                    "batch handled, updating local iterator"
+                );
+                // Update the iterator _after we've handled all the messages in the batch_.
+                iterator = Some(resp.iterator.clone());
+                // If the iterator is "done" we can backoff to wait for new messages to arrive.
+                sleep_time = if resp.done {
+                    // BACKOFF
+                    (sleep_time * 2).clamp(MIN_SLEEP, MAX_SLEEP)
+                } else {
+                    NO_SLEEP
+                };
+            }
+
+            Err(err) => {
+                tracing::trace!(
+                    error =? err,
+                    iterator =? &iterator,
+                    "request failed, retrying current iterator"
+                );
```

Comment:
Since you updated the other two, how about doing this one as well :smile: 

```suggestion
                tracing::trace!(
                    error = ?err,
                    ?iterator,
                    "request failed, retrying current iterator"
                );
```